### DISCUSSION TITLE
Background blur

### DISCUSSION THREAD

**Discussion by bm55** [damongolding/immich-kiosk#169] [General]
Is background blur working for you? I have it configured in the config.yml file and it was working but now it's not working. 
I am running 0.12.0

```
# UI
disable_ui: false # this is just a shortcut for all ui elements (show_time, show_date, show_image_time, show_image_date)
hide_cursor: false # Hide cursor/mouse via CSS.
font_size: 100 # the base font size as a percentage. OMIT the % character, default 100
background_blur: true # display a blurred version of image as background
theme: fade # which theme to use. fade or solid
layout: single # which layout to use. single or splitview
```

**Comment by damongolding** [comment]
What do you see on the front end?

Any errors in the Kiosk log?

**Reply by bm55** [reply]
the front is showing the images with black background instead of the blur.
The docker logs shows following. I don't have the album id from the logs  configured 

`Version 0.12.0


Kiosk listening on port 3000

16:26:10 ERRO no images found for album=c2dffec5-87a5-410b-9bbb-cf6af98ce92e
16:26:10 ERRO prefetch err="selecting image: no images found for album c2dffec5-87a5-410b-9bbb-cf6af98ce92e"
19:09:35 ERRO no images found for album=73c3706c-0c3f-40b8-bb37-52fc28460104
19:09:35 ERRO retrieving image err="selecting image: no images found for album 73c3706c-0c3f-40b8-bb37-52fc28460104"
20:48:15 ERRO no images found for album=73c3706c-0c3f-40b8-bb37-52fc28460104
20:48:15 ERRO retrieving image err="selecting image: no images found for album 73c3706c-0c3f-40b8-bb37-52fc28460104"
20:48:32 ERRO no images found for album=73c3706c-0c3f-40b8-bb37-52fc28460104
20:48:32 ERRO prefetch err="selecting image: no images found for album 73c3706c-0c3f-40b8-bb37-52fc28460104"
20:49:36 ERRO no images found for album=73c3706c-0c3f-40b8-bb37-52fc28460104
20:49:36 ERRO prefetch err="selecting image: no images found for album 73c3706c-0c3f-40b8-bb37-52fc28460104"
20:50:25 INFO Cache before flush viewDataCache_items=8 apiCache_items=44
20:50:25 INFO Cache after flush  viewDataCache_items=0 apiCache_items=0
20:52:55 ERRO unexpected status code: 404
20:52:55 ERRO prefetch err="selecting image: getting image preview: unexpected status code: 404"
20:54:37 ERRO no images found for album=166f3d37-48d0-470e-8257-4e64cc7250f1
20:54:37 ERRO prefetch err="selecting image: no images found for album 166f3d37-48d0-470e-8257-4e64cc7250f1"
20:57:49 ERRO no images found for album=166f3d37-48d0-470e-8257-4e64cc7250f1
20:57:49 ERRO prefetch err="selecting image: no images found for album 166f3d37-48d0-470e-8257-4e64cc7250f1"
21:00:21 ERRO no images found for album=73c3706c-0c3f-40b8-bb37-52fc28460104
21:00:21 ERRO prefetch err="selecting image: no images found for album 73c3706c-0c3f-40b8-bb37-52fc28460104"
`
There is only 1 album configured on my config file and no id is passes via URL
`
album: # ID(s) of album or albums to display
  - "d1c5091d-f004-4a6d-bff4-e20af812ac76" 

`

**Reply by bm55** [reply]
ok, looks like those albums in the docker logs are video albums without any photos and I do have one device using URL parameter "album=all"

**Reply by damongolding** [reply]
> ok, looks like those albums in the docker logs are video albums without any photos and I do have one device using URL parameter "album=all"

Yeah that will be what those errors are.

**Comment by damongolding** [comment]
You could add verbose debugging to your config which will show lots of debugging data.

```yaml
kiosk:
  debug_verbose: true
```

What you are looking for is something like this 👇
![image](https://github.com/user-attachments/assets/54ca3ef5-c0f0-408c-a74a-775572ad73af)

The `Blurred image in=0.168457458` is saying the image was blurred in 0.168457458 seconds, so I at least know it's creating the blurred background.

**Reply by bm55** [reply]
The debug only shows "converted image"
<img width="276" alt="image" src="https://github.com/user-attachments/assets/de30ef1f-f49a-4bf7-8538-5a0d100aad1b">
<img width="184" alt="image" src="https://github.com/user-attachments/assets/5ee0d663-d2f9-4e5f-b504-126e05e3e955">


<img width="113" alt="image" src="https://github.com/user-attachments/assets/189f354d-c798-49e1-9176-5aab651f1e9a">

**Reply by damongolding** [reply]
Ahh you have smart-zoom enabled. It enables cover mode and disables the blurred background (as you shouldn't be able to see the background). https://github.com/damongolding/immich-kiosk?tab=readme-ov-file#smart-zoom
